.ignore

```diff
@@ -0,0 +1,12 @@
+# Sample ignore file
```

Comment:
I don't think you should have an ignore file in the project itself. What I suggest is to generate a sample ignore file `sample.ignore` inside `.squirrel` folder.